integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -439,18 +439,11 @@ fuse:
   user: 0
   group: 0
   fsGroup: 0
-  # fuse args specific to each standalone fuse
-  args:
-    - fuse
-    - --fuse-opts=allow_other
```

Comment:
This section is directly copied into the yaml file. From now everything (other than `- fuse`) inside the `args` is configurable, so we don't need the section anymore.